jujugui/static/gui/src/app/components/unit-list/unit-list.js

```diff
@@ -205,25 +218,48 @@ YUI.add('unit-list', function() {
     },
 
     /**
+      Update the count of the number of active checkboxes.
+
+      @method _updateActiveCount
+    */
+    _updateActiveCount: function() {
+      var activeCount = 0;
+      var refs = this.refs;
+      Object.keys(refs).forEach((ref) => {
+        if (ref.split('-')[0] === 'UnitListItem') {
+          if (refs[ref].state.checked) {
+            activeCount += 1;
+          }
+        }
+      });
+      this.setState({'activeCount': activeCount});
```

Comment:
I like how you generate this count every time one of the children changes instead of trying to maintain an internal state of the children. Probably not as scalable (maybe we'll have to revisit) but less error prone imho.